Xamarin.Essentials/Calendars/Calendars.android.cs

```diff
@@ -636,8 +639,8 @@ static RecurrenceRule GetRecurranceRuleForEvent(string rule)
             {
                 var ruleDayIterationOffset = rule.Substring(rule.IndexOf("BYSETPOS=", StringComparison.Ordinal) + 9);
                 ruleDayIterationOffset = ruleDayIterationOffset.Contains(";") ? ruleDayIterationOffset.Substring(0, ruleDayIterationOffset.IndexOf(";", StringComparison.Ordinal)) : ruleDayIterationOffset;
-                recurranceRule.WeekOfMonth = (IterationOffset)Convert.ToInt32(ruleDayIterationOffset.Split(',').FirstOrDefault());
-
+                int.TryParse(ruleDayIterationOffset.Split(',').FirstOrDefault(), out var result);
```

Comment:
Should we be doing Enum.parse here if we're casing to IterationOffset?

Reply:
Altered accordingly